### PR TITLE
feat: support for unmanaged snyk-to-html

### PR DESCRIPTION
**Pull request by dagrest**
- [ ] Follows [CONTRIBUTING](https://github.com/snyk/snyk/blob/master/CONTRIBUTING.md) rules

#### What does this PR do?
Add support of snyk-to-html for snyk test --unmanaged --json

#### Where should the reviewer start?
https://github.com/snyk/snyk-to-html/blob/master/README.md

#### How should this be manually tested?
Run the following commands in the folder of unmanaged project/repo:
- to create vulnerability test for unmanaged project/repo:
path_to_git/cli/bin/snyk test --json --unmanaged | path_to_git/snyk-to-html/dist/index.js -o result.json
- to build HTML report of received test resluts:
path_to_git/snyk-to-html/dist/index.js -i result.json -o result.html

#### Any background context you want to provide?
[snyk code for snyk-to-html](https://www.notion.so/snyk/snyk-code-for-snyk-to-html-9231a0bd986e42fab61780eeca476998#8b1663e9273240a4aee23f72a272a2b4)

#### What are the relevant tickets?
[Jira ticket TUN-27](https://snyksec.atlassian.net/browse/TUN-27)

#### Screenshots
image

### CODE REVIEW

File: src/lib/ecosystems/resolve-test-facts.ts [hunk 1]
```diff
@@ -45,12 +46,35 @@ export async function resolveAndTestFacts(
           policy,
         );
 
+        const issuesDict: { [id: string]: Issue } = {};
```
Comment (myarichuk) [inline src/lib/ecosystems/resolve-test-facts.ts:49]: Judging by the name, I think you intended to use some sort of a Hashmap, right? 
If so, Typescript has [Map<Key,Value>](https://howtodoinjava.com/typescript/maps/) types that can be used here.

File: src/lib/ecosystems/resolve-test-facts.ts [hunk 2]
```diff
@@ -45,12 +46,35 @@ export async function resolveAndTestFacts(
...
+        response.issues.forEach(function(i) {
```
Comment (myarichuk) [inline src/lib/ecosystems/resolve-test-facts.ts:50]: By the way, you don't have to use JS anonymous function syntax, you can use [arrow functions ](https://www.tutorialsteacher.com/typescript/arrow-function)too!

**Review by myarichuk — APPROVED** [review]
lgtm, but take a look at my comments :)

**Review by danielroymoore — APPROVED** [review]
LGTM after comments resolved.

**Pull request merged**